src/tray_icon.rs

```diff
@@ -0,0 +1,43 @@
+use std::sync::mpsc;
+
+use crate::Icon;
+
+pub struct TrayIcon {
+    app: systray::Application,
+}
+
+impl TrayIcon {
+    pub fn new(tx: mpsc::Sender<Option<(usize, usize)>>) -> Result<Self, ()> {
+        let mut icon = match systray::Application::new() {
+            Ok(app) => Ok(Self { app }),
+            Err(e) => {
+                println!("Could not create gtk application: {}", e);
```

Comment:
While we're moving these around, let's make them also output to the right output stream.
```suggestion
                eprintln!("Could not create gtk application: {}", e);
```